src/data/rat-partner.json

```diff
@@ -0,0 +1,89 @@
+                }
+            }
+    },
+    "section-contribute-listing": {
+        "content": {
+            "feature-list": [{
+                "image": {
+                    "title": "Questions",
+                    "url": "/assets/img/icons/questions.svg",
+                    "alt": "Questions icon"
+                },
+                "title": "Frequently asked questions",
+                "text": "Frequently asked questions are answered in the <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki/FAQ' target='_blank' rel='noopener noreferrer' title=''>FAQ on GitHub</a>."
+            },{
+                "image": {
+                    "title": "Improvement",
+                    "url": "/assets/img/icons/improvement.svg",
+                    "alt": "Improvement icon"
+                },
+                "title": "Technical issues",
+                "text": "In case of technical issues during the test phase, please use this e-mail address: <a href='mailto:cwa-schnelltest-onboarding@t-systems.com' target='_blank' rel='noopener noreferrer' title=''>cwa-schnelltest-onboarding@t-systems.com</a>."
+            },{
+                "image": {
+                    "title": "Documentation",
+                    "url": "/assets/img/icons/documentation.svg",
+                    "alt": "Documentation icon"
+                },
+                "title": "Conclusion of the contract",
+                "text": "Prerequisite for the connection to the Corona-Warn-App is a signed contract, <a href='https://github.com/corona-warn-app/cwa-quicktest-onboarding/wiki/Vertragsabschluss' target='_blank' rel='noopener noreferrer' title=''>details can be found here</a>. The contracts are available on request by e-mail to <a href='mailto:registrierung.labore.pandemietest@t-systems.com' target='_blank' rel='noopener noreferrer' title=''>registrierung.labore.pandemietest@t-systems.com</a>. Due to the high number of inquiries, there may currently be delays in responding."
+            }]
+        },
+        "box": {
+            "class": "wide",
+             "image": {
+                    "url": "/assets/img/ideas.svg",
+                    "title": "Ideen",
+                    "alt": "Ideen-icon"
+            },
+            "content": {
+                "title": "Would you like to contact us?",
+                "text": "If you have any further questions, please send us an e-mail: <a href='mailto:registrierung.labore.pandemietest@t-systems.com' target='_blank' rel='noopener noreferrer' title=''>registrierung.labore.pandemietest@t-systems.com</a>. We look forward to hearing from you!"
+            }
+        }
+    },
+    "section-start-contribute": {
+        "headline":{
+            "title": "Join now!",
+            "button": {
+                "title": "Become a rapid test partner",
```

Comment:
I think we should not use "Become a rapid test partner" here again, here's why:

The user gets hand offed to the site `rat-partner` by clicking a button on the main page which has exactly the same text. Users might think that this button does nothing (or that the page will just jump to the top) because they don't expect a button with the same text to do different things.

Reply:
Sounds good to me!